src/phoenix/datasets/experiments.py

```diff
@@ -460,13 +483,53 @@ async def async_evaluate_run(
         fallback_return_value=None,
         tqdm_bar_format=get_tqdm_progress_bar_formatter("running experiment evaluations"),
     )
-    evaluation_payloads, _execution_details = executor.run(evaluation_inputs)
+    evaluation_payloads, _execution_details = executor.run(evaluation_input)
     for payload in evaluation_payloads:
         if payload is not None:
             resp = client.post("/v1/experiment_evaluations", json=jsonify(payload))
             resp.raise_for_status()
 
 
+class _Surrogate(SurrogateEvaluator): ...
```

Comment:
What is the purpose of the subclass here?

Reply:
This is just a "decorator" to unify both raw functions and Evaluator class

Reply:
Oh, this is to make it a private variable to indicate that users shouldn't be using this. (If they do use the public version will not instantiate.)

Reply:
No, but because it's in its own module, it's kind of public in that way

Reply:
I'll probably just remove it and use Dustin's decorators

Reply:
Okay.